### ISSUE TITLE
How to use DefaultLogger in web services?

### ISSUE THREAD

**Issue by Amargin** [micro/go-micro#1514]
Hi, thanks for great framework! 
I want to use custom logger into go-micro web service to get the same format across application. It works well with srv services, but web seems doesn't use default logger while initialisation.
```go
package main

import (
	"github.com/micro/go-micro/v2"
	"github.com/micro/go-micro/v2/logger"
	"github.com/micro/go-micro/v2/web"
	microzap "github.com/micro/go-plugins/logger/zap/v2"
	"go.uber.org/zap"
)

func main() {
	z, _ := microzap.NewLogger(microzap.WithConfig(zap.NewProductionConfig()))
	logger.DefaultLogger = z
	logger.Info("log message")

	web := web.NewService()
	web.Init()
	go web.Run()

	micro := micro.NewService()
	micro.Init()
	micro.Run()
}
```
```
{"level":"info","ts":1586436287.027182,"caller":"logger/level.go:70","msg":"log message"}
{"level":"info","ts":1586436287.0286322,"caller":"logger/level.go:74","msg":"Starting [service] go.micro.server"}
{"level":"info","ts":1586436287.028746,"caller":"logger/level.go:74","msg":"Server [grpc] Listening on [::]:60345"}
{"level":"info","ts":1586436287.028982,"caller":"logger/level.go:74","msg":"Registry [mdns] Registering node: go.micro.server-f5f02e04-e6d8-4ec5-8f5f-259146a51907"}
2020-04-09 16:44:47  level=info service=web Listening on [::]:60344
```
Possible reason is using https://github.com/micro/go-micro/blob/master/web/web.go#L42 instead of default logger in https://github.com/micro/go-micro/blob/master/web/service.go#L230?

**Comment by vtolstov** [comment]
in near feature we re-apply logger to all micro objects and use global default if it not passed.

**Comment by vtolstov** [comment]
fixed in #1577

**Comment by FlyingOnion** [comment]
Hi @vtolstov , I think the logger of server [here](https://github.com/micro/go-micro/blob/master/server/server.go#L152) could be modified like this too.
I also have a question. Currently the logger is not integrated into the framework. Each module has their own loggers, but some of them could not be customized (like web and server). When I use zap as the logger, I found that using bare zap without implementing the `Logger` interface has little influence (although the logs differ). IMO it is better adding the logger into options of each module (maybe in the next version). Please let me know your opinion, and what I can do to contribute. Thanks.
